servicenow/icon_servicenow/triggers/incident_created/trigger.py

```diff
@@ -0,0 +1,68 @@
+import komand
+import time
+from .schema import IncidentCreatedInput, IncidentCreatedOutput, Input, Output, Component
+# Custom imports below
+
+
+class IncidentCreated(komand.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='incident_created',
+                description=Component.DESCRIPTION,
+                input=IncidentCreatedInput(),
+                output=IncidentCreatedOutput())
+        self.found = {}
+        self.url = ""
+        self.method = "get"
+        self.query = ""
+
+    def initialize(self):
+        if self.query:
+            response = self.connection.request.make_request(
+                self.url, self.method, params=self.query)
+        else:
+            response = self.connection.request.make_request(self.url, self.method)
+
+        try:
+            results = response.get("resource").get("result")
+        except KeyError as e:
+            raise PluginException(preset=PluginException.Preset.UNKNOWN,
+                                  data=response.text) from e
+
+        sys_ids = [result.get("sys_id") for result in results]
```

Comment:
```suggestion
        sys_ids = [result.get("sys_id") for result in results if result.get("sys_id") is not None]
```
Need to ensure that None values don't pass into your loop iterations below if `sys_id` is missing from the response